db/schema.rb

```diff
@@ -113,6 +119,17 @@
     t.index ["streak_key"], name: "index_letters_on_streak_key", using: :btree
   end
 
+  create_table "slack_users", force: :cascade do |t|
+    t.integer  "user_id"
+    t.text     "slack_uid"
+    t.text     "team_id"
+    t.text     "scopes",                    array: true
+    t.text     "access_token"
+    t.datetime "created_at",   null: false
+    t.datetime "updated_at",   null: false
+    t.index ["user_id"], name: "index_slack_users_on_user_id", using: :btree
+  end
+
```

Comment:
@zachlatta: Which specific lines should I get rid of?

I'm assuming:

- 129
- 142
- 143